main/zoe/guide/contracts/loan.md

```diff
@@ -0,0 +1,143 @@
+# Loan
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts/loan)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+The basic loan contract allows a  borrower to add collateral of a
+particular brand and get a loan of another brand. The collateral (also
+known as margin) must be greater than the loan value, at an amount set
+by the Maintenance Margin Requirement (`mmr`) in the terms of the
+contract.
+
+The loan does not have a distinct end time. Rather, if the
+value of the collateral changes such that insufficient margin is
+provided, the collateral is liquidated, and the loan is closed. At any
+time, the borrower can add collateral or repay the loan with interest,
+closing the loan. The borrower can set up their own margin calls by
+getting the [`priceAuthority`](/zoe/guide/price-authority.md) from the terms and calling
+`E(priceAuthority).quoteWhenLT(allCollateral, x)` where x is the value
+of the collateral in the Loan brand at which they want a reminder to
+addCollateral.
+
+Note that all collateral must be of the same brand and all of the
+loaned amount and interest must be of the same (separate) brand.
+
+## Terms
+* `mmr` (default = 150) - the Maintenance Margin Requirement, in
+   percent. The default is 150, meaning that collateral should be
+   worth at least 150% of the loan. If the value of the collateral
+   drops below `mmr`, liquidation occurs.
+* [`priceAuthority`](/zoe/guide/price-authority.md) - will be used for getting the current value of
+   collateral and setting liquidation triggers.
+* `autoswapInstance` - The running contract instance for an
+   [Autoswap](./autoswap.md) or [Multipool
+   Autoswap](./multipoolAutoswap.md) installation. The `publicFacet`
+   of the instance is used for producing an invitation to sell the
+   collateral on liquidation.
```

Comment:
```suggestion
   Autoswap](./multipoolAutoswap.md) installation. The instance's `publicFacet`
   produces an invitation to sell the collateral on liquidation.
```

Reply:
This sounds like it is active, but the passive voice is more appropriate because we are calling a method on the publicFacet.

Reply:
Revised